.github/workflows/ci-test.yml

```diff
@@ -125,6 +144,314 @@ jobs:
           exit $success
         env:
           DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE: ${{ secrets.DOCKERHUB_KEY_PASSWORD }}
+      - name: push to novi ecr
+        run: |
+          #push to novi ecr with standard names
+          BRANCH="$CHANGES_TARGET_BRANCH"
+          GIT_REV=$(git rev-parse --short=8 HEAD)
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | \
+          docker login --username AWS --password-stdin "${AWS_ECR_ACCOUNT_URL}"
+          docker/docker_republish.sh -t pre_${BRANCH}_${GIT_REV} -o land_${GIT_REV} -r ${AWS_ECR_ACCOUNT_URL} -d
+
+  land-blocking-test:
+    name: Run land blocking test
+    runs-on: self-hosted
+    needs: build-images
+    # NOTE the total time should cover build (~10 min) and test (~20 min).
+    # The additional time can cover the retries and wait.
+    timeout-minutes: 60
```

Comment:
Will need to sample/eye-ball a few current tests to determine what the new `timeout-minutes` would be.

BTW, the comment looks outdated, as the build takes place in the prior job.

Reply:
it takes around 25~28 mins based on canary I have done. I will put 30 here first and will take a look for next few builds

Reply:
not great to be at 45 minutes or so for lbt (docker build + lbt that is).  we may need to improve the build time.  I can suggest a few ways.   Or take them on.